doc/lsp.txt

```diff
@@ -444,14 +444,20 @@ snippetSupport		|Boolean| option.  Enable snippet completion support.
 			Need a snippet completion plugin like vim-vsnip.
 			By default this is set to false.
 						*lsp-opt-ultisnipsSupport*
-ultisnipsSupport		|Boolean| option.  Enable SirVer/ultisnips support.
+ultisnipsSupport	|Boolean| option.  Enable SirVer/ultisnips support.
 			Need a snippet completion plugin SirVer/ultisnips.
 			By default this is set to false.
 						*lsp-opt-usePopupInCodeAction*
 usePopupInCodeAction    |Boolean| option.  When using the |:LspCodeAction|
 			command to display the code action for the current
 			line, use a popup menu instead of echoing.
 			By default this is set to false.
+						*lsp-opt-customCompletionKinds*
+customCompletionKinds   |Boolean| option.  If you set this to true, you can set
+			custom completion kinds using the option completionKinds.
+						*lsp-opt-completionKinds*
+completionKinds		|Dictionary| option. Here you can set custom completion
+			kinds. To view all completion kind names, use :h lsp-custom-kinds
```

Comment:
I think that if you change it to `|lsp-custom-kinds|` then you can jump to it directly from the help page. `See |lsp-custom-kinds| for all completion kind names`.

Reply:
Ok! I'll change it now

Reply:
Done.